lib/Organizations.php

```diff
@@ -65,7 +65,8 @@ public function listOrganizations(
      * Create Organization.
      *
      * @param string $name The name of the Organization.
-     * @param array $domains The domains of the Organization.
+     * @param null|array $domains [Deprecated] The domains of the Organization. Use domain_data instead.
```

Comment:
We should add something along the lines of "Will be removed in a future major version" to further encourage people not to use this.

Reply:
That's generally what "Deprecated" means to me, but no harm in adding some more context if you think it'd be useful.

Reply:
I find it better to be extra explicit in these cases, not a big deal though.